evennia/scripts/globalhandler.py

```diff
@@ -0,0 +1,48 @@
+from django.conf import settings
+from evennia.utils.utils import class_from_module
+
+
+class GlobalHandler(object):
+    """
+    Simple Handler object loaded by the Evennia API to contain and manage a game's Global Scripts.
+
+    This is accessed like a dictionary. Alternatively you can access Properties on it.
+
+    Example:
+        import evennia
+        evennia.GLOBAL_SCRIPTS['key']
```

Comment:
The use of the property should be the recommended way and using 'get' if not knowing the name beforehand. It makes the API a lot nicer. This use of a dict-like call matches no other handler in Evennia.